docs/protocol.md

```diff
@@ -21,14 +21,24 @@ Currently used resources are: `catalog`, `meta`, `stream`, `subtitles`, `watchSt
 
 `/subtitles/{type}/{id}.json` - list of all subtitles for a particular item; `type` again denotes the type, the `id` in this case is the Open Subtitles file hash, while `extraArgs` (read below) is used for `videoID` (the ID of the particular item, as found in the catalog or a video ID) and `videoSize` (video file size in bytes)
 
-`/player/{type}/{id}/{extraArgs}.json` - TODO
+the `player` and `library` resources are events. in other resources the addon is supposed to provide data. but in event resources the addon recieves data and is expected to process that given data then respond with the success or failure of that processing.
+
+`/player/{type}/{videoID}/{extraArgs}.json` - a player event; `type` again denotes the type, and `videoID` is the video ID
 
 Where `extraArgs` can be one of:
 - Play: `action=play&currentTime={milliseconds}&duration={milliseconds}`
 - Start: `action=start&currentTime={milliseconds}&duration={milliseconds}`
 - End: `action=end&currentTime={milliseconds}&duration={milliseconds}`
 - Pause: `action=pause&currentTime={milliseconds}&duration={milliseconds}`
 
+`/library/{type}/{id}/{extraArgs}.json` - a library event; `type` again denotes the type, and `id` is the ID of the particular item
+
+Where `extraArgs` can be one of:
+- Add to library: `action=libraryAdd`
+- Remove from library: `action=libraryRemove`
```

Comment:
I suggest to rename these to `add` and `remove` respectively.
We are already under the `library` resource and a simple action  name would be sufficient